drivers/sensor/icm42670S/imu/inv_imu_driver.c

```diff
@@ -1483,6 +1483,10 @@ static int configure_serial_interface(inv_imu_device_t *s)
 			return INV_ERROR_BAD_ARG; /* Not supported */
 		status |= inv_imu_write_reg(s, DEVICE_CONFIG, 1, &value);
 
+		/* Workaround : Overwrite DRIVE_CONFIG3 register with 0x4 value */
```

Comment:
Do we want to emphasize publicly that the IMU has a bug?

Reply:
Could we address this point prior to open the PR to zephyr? I would merge it as it fixes the SPI functionality. This way we could move forward with more testing/formatting/...

Reply:
Ok, I removed the workaround